Student-Unity/src/main/java/csed/swe/studentunity/model/RegisteredCourseId.java

```diff
@@ -11,27 +10,27 @@
 @Setter
 public class RegisteredCourseId implements Serializable {
 
-    private Long courseId;
-    private Long userId;
```

Comment:
Why you have changed the userId to user ???
I now that it is user type but we are only interested in its id and we do so in the contractor

Reply:
I thought so at the beginning so I used the id only like you did in the task, but the problem is if the user got deleted from the users table his registered courses should be deleted too, so we need a foreign key, is the right way of making a foreign key using the long id only or referencing the user object through mapping and "refrencedColumnName"?

Reply:
to be able to reference the user PK from ``RegisteredCourses`` Entitiy